pkg/storage/wrapper/prefix/prefix_wrapper.go

```diff
@@ -0,0 +1,76 @@
+/*
+Copyright SecureKey Technologies Inc. All Rights Reserved.
+SPDX-License-Identifier: Apache-2.0
+*/
+
+package prefix
+
+import (
+	"fmt"
+
+	"github.com/hyperledger/aries-framework-go/pkg/storage"
+)
+
+// package IDPrefix offers provider.Store wrappers for stores with certain constraints on key values such as
+// reserved IDPrefix values (eg key IDs starting with underscore) by prefixing key values.
+
+// IDPrefix is the prefix added to KIDs in StorePrefixWrapper.
+const IDPrefix = "p"
+
+// NewPrefixStoreWrapper creates a new StorePrefixWrapper of store.
+func NewPrefixStoreWrapper(store storage.Store) storage.Store {
```

Comment:
Do not return interface, please return a structure.
We need to follow the simple rule `Accept interfaces and return structs`.

Reply:
updated